sonaranalyzer-dotnet/SonarAnalyzer.sln

```diff
@@ -45,6 +45,20 @@ Project("{FAE04EC0-301F-11D3-BF4B-00C04F79EFBC}") = "CBDESucceedsWithIncorrectRe
 EndProject
 Project("{FAE04EC0-301F-11D3-BF4B-00C04F79EFBC}") = "CBDEArguments", "tests\CBDE\CBDEArguments\CBDEArguments.csproj", "{037E7106-8196-4D7C-A532-72730C304CB7}"
 EndProject
+Project("{9A19103F-16F7-4668-BE54-9A1E7A4F7556}") = "mscorlib", "tests\XXE\Mscorlib3.5Mock\mscorlib.csproj", "{373C95E5-0A57-4494-8E5C-3CE8BF7D6E60}"
+EndProject
+Project("{9A19103F-16F7-4668-BE54-9A1E7A4F7556}") = "mscorlib", "tests\XXE\Mscorlib4.0Mock\mscorlib.csproj", "{684F961D-87E3-44F8-82BF-29BD71E130A3}"
+EndProject
+Project("{9A19103F-16F7-4668-BE54-9A1E7A4F7556}") = "mscorlib", "tests\XXE\Mscorlib4.0MockWithIo\mscorlib.csproj", "{614BAED6-8F6B-48E4-A3CD-E34851516A37}"
+EndProject
+Project("{2150E333-8FDC-42A3-9474-1A3956D46DE8}") = "XXE_mocks", "XXE_mocks", "{6544F153-B0A5-4923-A2E0-E8F536DE18A1}"
+EndProject
+Project("{2150E333-8FDC-42A3-9474-1A3956D46DE8}") = "3.5", "3.5", "{9BFBA760-D08C-4A13-9463-4FAC3CFBA97F}"
+EndProject
+Project("{2150E333-8FDC-42A3-9474-1A3956D46DE8}") = "4.0_no_IO", "4.0_no_IO", "{305F8BA2-C772-44D5-9EA4-A556ED3B8F7E}"
+EndProject
+Project("{2150E333-8FDC-42A3-9474-1A3956D46DE8}") = "4.0_with_IO", "4.0_with_IO", "{50657C7A-950C-4855-9E30-53262D523766}"

```

Comment:
- one doesn't have `UnmanagedMemoryStream` at all
- one has `UnmanagedMemoryStream` but doesn't have `FlushAsync` member (so it could be 4.0 or 4.5.1)

so both are not 4.5.2